fiftyone/core/models.py

```
@@ -1016,6 +896,91 @@ def _make_data_loader(
     )
 
 
+class _VideoCollateFn:
+    def __init__(self, model_collate):
```

Ruff (0.15.12)

[warning] Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)